### PR TITLE
Docs to make theme command launch with content

### PR DESCRIPTION
**Pull request by pvpg**
- Added documents for the theme command
- Changed the README and package.json description/info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,20 @@
-## Sample Blog with API-based CMS
+# Prismic & Node.js Example Blog
 
-Node.js blog is a blog template with content managed in prismic.io (an API-based CMS).
+[Node.js](https://www.php.net/) example blog project with content managed in [Prismic](https://prismic.io/)
```
Comment (levimykel) [inline README.md]: Incorrect link

**Comment by levimykel** [comment]
Also did you test all of these to make sure everything is working?

File: README.md (hunk 1 above)
Comment (pvpg) [reply README.md]: done

**Comment by pvpg** [comment]
Yes, I tested them all before opening the PRs

**Pull request merged**